### DISCUSSION TITLE
Should parse to link if a autolink chunk got a prefix mark?

### DISCUSSION THREAD

**Discussion by zzuu666** [remarkjs/remark#678] [General] [answered]
example:
``` md
,https://github.com
```

there is a autolink chunk `https://google.com`, but it got a prefix mark `,`
in remark 13.0 with `remark-gfm` plugin, the result after be parsed as follow image:
![image](https://user-images.githubusercontent.com/11661645/113989508-15dee900-9883-11eb-98b4-a48841859945.png)

but, in most markdown interpreter(include here...) would like to parse this chunk to  ,https://github.com

**Comment by wooorm** [comment, marked as answer]
Are you sure remark-gfm is used on astexplorer?

**Reply by zzuu666** [reply]
Yes, I'm sure.
And, you can see an example in here https://codesandbox.io/s/modest-leaf-6j1k6?file=/src/index.js

**Reply by wooorm** [reply]
nice catch.

this is another example of the two GH algorithms being at odds:

See this text:

```markdown
,https://github.com

[ ,https://github.com

[asd] ,https://github.com
```

Rendered on issues:

,https://github.com

[ ,https://github.com

[asd] ,https://github.com

And rendered in a readme: https://gist.github.com/wooorm/b4fde644e06b19591ec3176228b3fed8

**Reply by wooorm** [reply]
this is working as expected in `micromark-extension-gfm-autolink-literal`

**Reply by wooorm** [reply]
It’s also working in https://github.com/syntax-tree/mdast-util-gfm-autolink-literal.


Maybe you’re on an older release, `npm update` should do the trick

**Reply by zzuu666** [reply]
> this is working as expected in `micromark-extension-gfm-autolink-literal`

Thx, let me have a try.
